src/main/kotlin/racingcar/RacingCarMain.kt

```diff
@@ -0,0 +1,30 @@
+package racingcar
+
+import racingcar.RacingCar.Companion.MAXIMUM_POWER_NUMBER
+import racingcar.RacingCar.Companion.MINIMUM_POWER_NUMBER
+
+fun main() {
+    RacingCarView.printCarsInputView()
+    val carCount = getIntValue()
+    RacingCarView.printAttemptInputView()
+    var attemptNumber = getIntValue()
+    val cars = mutableListOf<RacingCar>()
+    for (i in 0 until carCount) {
+        cars.add(RacingCar())
+    }
+    RacingCarView.printResultView()
+    while (attemptNumber-- > 0) {
+        runCars(cars)
+        RacingCarView.printCarsPosition(cars)
+    }
+}
+
+fun getIntValue(): Int {
+    val input = readlnOrNull()
```

Comment:
```suggestion
    val input = readln()
```

readln()을 사용하여 null을 방지하는 건 어떨까요?

참고 
- [kotlin readln](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.io/readln.html)
- [kotlin 1.6.0 released](https://blog.jetbrains.com/ko/kotlin/2021/12/kotlin-1-6-0-is-released/)